cms/templates/partials/faculty-carousel.html

```diff
@@ -0,0 +1,26 @@
+{% load wagtailcore_tags image_version_url %}
```

Comment:
Why is this names faculty-**carousel**.html? The design does not look like a carousel.